kite-tools-parent/pom.xml

```diff
@@ -180,6 +180,12 @@
       <scope>compile</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.kamranzafar</groupId>
```

Comment:
I've used this one before without any issues but should definitely use existing Apache functionality if possible. I'll play with the stuff that came over from ant into the commons compress lib.